.github/workflows/strr-application-auto-approval-job-cd.yaml

```diff
@@ -0,0 +1,30 @@
+name: STRR APPLICATION AUTO APPROVAL JOB CD
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "jobs/auto-approval/**"
```

Comment:
if the job uses api binary, need to also include path so api, so that rebuild is triggered, e.g. https://github.com/bcgov/sbc-auth/blob/78dbf88a073afac6ec0218b7e0ce638f03e87867/.github/workflows/auth-queue-cd.yml#L8

Reply:
i have added the paths